src/main/java/ContentOutput.java

```diff
@@ -0,0 +1,12 @@
+import java.io.IOException;
+/**
+ * Interface for writing content into source
+ */
+public interface ContentOutput {
+    /**
+     * Writing content into source
```

Comment:
@ArthurRomasyuk The doc should be answering the question `What it does?` not `What is it doing?`